doc/contributing/testing_ruby.md

```diff
@@ -20,10 +20,11 @@ We can run any of the make scripts [in parallel](building_ruby.md#label-Running+
     make btest OPTS=-v
     ```
 
-    To run an individual bootstrap test, we can set the filename in the environment variable `BTESTS`:
+    To run individual bootstrap tests, we can either specify a list of filenames or use the `--sets` flag in the environment variable `BTESTS`:
```

Comment:
```suggestion
    To run individual bootstrap tests, we can either specify a list of filenames or use the `--sets` flag in the variable `BTESTS`:
```

Reply:
https://www.gnu.org/software/make/manual/make.html#Reference says `variables`

Reply:
I think this is just an environment variable. I don't think we should be using GNU make specific concepts, it would be more confusing for the reader.

```
$ export BTESTS="--sets=fork,gc"
$ make btest

["test_fork.rb", "test_gc.rb"]
2022-07-29 09:30:07 -0400
Driver is ruby 3.1.2p20 (2022-04-12 revision 4491bb740a) [arm64-darwin21]
Target is ruby 3.2.0dev (2022-07-27T13:16:04Z master 70a9328b79) [arm64-darwin21]

test_fork.rb  PASS 4
test_gc.rb    PASS 2

Finished in 0.93 sec

PASS all 6 tests
```

Reply:
No, it just happens to work. The make variable works with lower-case. We didn't use lower-case value as environmental variables.

Reply:
Ah I see, I've updated it.